src/promptflow/promptflow/contracts/tool.py

```diff
@@ -52,7 +62,15 @@ def from_value(t: Any):
         return ValueType.OBJECT
 
     @staticmethod
-    def from_type(t: type):
+    def from_type(t: type) -> "ValueType":
+        """Get `ValueType` by type.
+
+        :param t: The type needs to get its `ValueType`
+        :type t: type
+        :return: The `ValueType` of the given type
+        :rtype: ValueType
```

Comment:
For these classes, use `~promptflow.xxx.ValueType`, it will be rendered to a link that points to the class definition

Reply:
Updated.